README.md

```diff
@@ -178,5 +184,14 @@ should produce the following output:
 
 Run `python setup.py test` from within the project directory.
 
+
+## Deployments
+Deployments to PyPi are handled through [Travis-CI](https://travis-ci.org/WikiWatershed/tr-55).  Tag a commit for release and it will be automatically published.
+
+``` bash
+git tag [tag name] [commit sha]
```

Comment:
For the Ansible roles, I generally use the following steps:

```bash
$ git flow release start 0.1.0
$ vim CHANGELOG.md
$ git commit -m "0.1.0"
$ git flow release publish 0.1.0 # If I want others to see it, or Travis to test it
$ git flow release finish 0.1.0
```

The `finish` step above handles merging the release branch into `develop` and `master`, as well as tag creation (but doesn't push anything remotely).

Does that process adjustment seem reasonable?